<?xml version="1.0"?>
<launch>
    <rosparam file="$(find unity_connector)/config/params.yaml" command="load"/>

    <arg name="actual_action_command_topic" default="/farmbot_connector/action"/>
	<arg name="actual_action_feedback_topic" default="/farmbot_connector/feedback"/>


    <node name="unity_connector_tcp_endpoint" pkg="unity_connector" type="ros_unity_tcp_endpoint.py" args="--wait" output="screen" respawn="true" >
		<param name="actual_action_feedback_topic"   value="$(arg actual_action_feedback_topic)"/>
		<param name="actual_action_command_topic"    value="$(arg actual_action_command_topic)"/>
    </node>
</launch>